src/webassembly/instance.rs

```diff
@@ -74,6 +74,81 @@ pub struct EmscriptenData {
     pub stack_alloc: extern "C" fn(u32, &Instance) -> u32,
 }
 
+impl EmscriptenData {
+    pub fn new(module: &Module, instance: &Instance) -> Self {
+        unsafe {
+            debug!("emscripten::new");
+            let malloc_export = module.info.exports.get("_malloc");
+            let free_export = module.info.exports.get("_free");
+            let memalign_export = module.info.exports.get("_memalign");
+            let memset_export = module.info.exports.get("_memset");
+            let stack_alloc_export = module.info.exports.get("stackAlloc");
+
+            let mut malloc_addr = 0 as *const u8;
```

Comment:
I think we should do something here instead of just assigning the function pointers to null, but that's probably for a later time.

Reply:
Yup! I will create a task for it so we don't forget in the future